pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -18,19 +18,51 @@ package volumebinding
 
 import (
 	"context"
+	"fmt"
+	"time"
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+	schedulerv1alpha2 "k8s.io/kube-scheduler/config/v1alpha2"
 	"k8s.io/kubernetes/pkg/controller/volume/scheduling"
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 )
 
+const (
+	// DefaultBindTimeoutSeconds defines the default bind timeout in seconds
+	DefaultBindTimeoutSeconds = 600
```

Comment:
`BindTimeoutSeconds = 100` in `scheduler.go` is moved here. I changed `100` to `600` (default in api types) because `100` in `scheduler.go` is never used because it will always be overridden by  `ComponentConfig.BindTimeoutSeconds`)

https://github.com/kubernetes/kubernetes/blob/be02fe578588592ee67811ed809eef0a4495537a/pkg/scheduler/apis/config/v1alpha2/defaults.go#L132-L135

https://github.com/kubernetes/kubernetes/blob/be02fe578588592ee67811ed809eef0a4495537a/cmd/kube-scheduler/app/server.go#L335-L337

Reply:
I see, sounds good.